responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -321,8 +257,83 @@ def _whatif(self, id, X, X_feature_new, feature_name, y, alpha=0.1):
                               y, alpha=alpha).to_dict(orient="records")
 
     def compute(self):
-        """No-op function to comply with model analysis design."""
-        pass
+        """Computes the causal effects by running the causal
+           configuration."""
+        is_classification = self._task_type == ModelTask.CLASSIFICATION
+        for result in self._results:
+            causal_config = result.config
+            if not result.is_computed:
```

Comment:
where is result.is_computed set to true?

Reply:
oh, nvm, I think I got confused - it looks like this isn't a bool property here but instead we are checking:
```
self.causal_analysis is not None or \
            self.global_effects is not None or \
            self.local_effects is not None or \
            self.policies is not None
```
I guess this is a little different from the other configs